.php-cs-fixer.dist.php

```diff
@@ -30,6 +30,7 @@
         '@Symfony:risky' => true,
         'protected_to_private' => false,
         'header_comment' => ['header' => $fileHeaderComment],
+        'native_function_invocation' => ['include' => ['@compiler_optimized', 'sprintf'], 'scope' => 'namespaced', 'strict' => true],
```

Comment:
I suggest adding a TODO to remove that line again once we use PHP-CS-Fixer 3.60+, as `@compiler_optimized` will include it